app/assets/javascripts/protocols_show.js.coffee

```diff
@@ -4,14 +4,11 @@ $ ->
 
     # Bootstrap Tab persistence
 
```

Comment:
The problem with the JS is that now, the page loads with no tab selected and it is very noticeable. I'm going to try moving logic to the controller